module.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mbed-hal-st-stm32f4",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
please don't change version within a pull request. You can write in the PR description - that this is a new minor version request